archiexpo.es,"HTTPSConnectionPool(host='archiexpo.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
archiexpo.it,"HTTPSConnectionPool(host='archiexpo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
architectureanddesign.com.au,"HTTPSConnectionPool(host='architectureanddesign.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.architectureanddesign.com.au', port=443): Read timed out. (read timeout=2)"
arcinfo.ch,"HTTPSConnectionPool(host='arcinfo.ch', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.arcinfo.ch', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
arcotampano.it,"HTTPSConnectionPool(host='arcotampano.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfb0a8bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='arcotampano.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfa5867f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
arda.or.th,"HTTPSConnectionPool(host='arda.or.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.arda.or.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
arenas-architecte.com,"HTTPSConnectionPool(host='arenas-architecte.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='arenas-architecte.com', port=443): Read timed out. (read timeout=2)"
argep.hu,"HTTPSConnectionPool(host='argep.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='argep.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
army.mil,"HTTPSConnectionPool(host='army.mil', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfb190250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='army.mil', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df949cbb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
arnes.si,"HTTPSConnectionPool(host='arnes.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df85e7130>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='arnes.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df93c8a00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
artesanatopassoapassoja.com.br,"HTTPSConnectionPool(host='artesanatopassoapassoja.com.br', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8dfa723970>, 'Connection to artesanatopassoapassoja.com.br timed out. (connect timeout=2)'))","HTTPConnectionPool(host='artesanatopassoapassoja.com.br', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8df8439460>, 'Connection to artesanatopassoapassoja.com.br timed out. (connect timeout=2)'))"
articles.co.il,"HTTPSConnectionPool(host='articles.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='articles.co.il', port=443): Read timed out. (read timeout=2)"
as-1.co.jp,"HTTPSConnectionPool(host='as-1.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df7ffa1f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='as-1.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df82c6340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
asamblea.gob.ni,"HTTPSConnectionPool(host='asamblea.gob.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df72cc820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='asamblea.gob.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df7a98220>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
asbeiras.pt,"HTTPSConnectionPool(host='www.asbeiras.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.asbeiras.pt', port=443): Read timed out. (read timeout=2)"
asfi.gob.bo,"HTTPSConnectionPool(host='asfi.gob.bo', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df88ef880>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='asfi.gob.bo', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df8012700>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
asklaila.com,"HTTPSConnectionPool(host='asklaila.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'asklaila.com' doesn't match '*.asklaila.com'"")))","HTTPSConnectionPool(host='asklaila.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'asklaila.com' doesn't match '*.asklaila.com'"")))"
askoservering.no,"HTTPSConnectionPool(host='askoservering.no', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='askoservering.no', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
asktel.ru,"HTTPSConnectionPool(host='asktel.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='asktel.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
aslimnica.lv,"HTTPSConnectionPool(host='aslimnica.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='aslimnica.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
assabah.com.tn,"HTTPSConnectionPool(host='assabah.com.tn', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfc52ef70>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='assabah.com.tn', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfa97fdc0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
assemblee-nationale.fr,"HTTPSConnectionPool(host='assemblee-nationale.fr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df64d1190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='assemblee-nationale.fr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df79240d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
astra.co.id,"HTTPSConnectionPool(host='www.astra.co.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.astra.co.id', port=443): Read timed out. (read timeout=2)"
asus.com,"HTTPSConnectionPool(host='asus.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df9fdc850>: Failed to establish a new connection: [Errno 111] Connection refused'))","('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))"
atauni.edu.tr,"HTTPSConnectionPool(host='atauni.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='atauni.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
atelier-povetron.cz,"HTTPSConnectionPool(host='atelier-povetron.cz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfc0363d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='atelier-povetron.cz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df662a190>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
atputasbazes.lv,"HTTPConnectionPool(host='atputasbazes.lv', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='atputasbazes.lv', port=80): Read timed out. (read timeout=2)"
auchan.ro,"HTTPSConnectionPool(host='www.auchan.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.auchan.ro', port=443): Read timed out. (read timeout=2)"
aun.edu.eg,"HTTPSConnectionPool(host='aun.edu.eg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8df8d52760>, 'Connection to aun.edu.eg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='aun.edu.eg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8dfad9e460>, 'Connection to aun.edu.eg timed out. (connect timeout=2)'))"
auone.jp,"HTTPSConnectionPool(host='auone.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='auone.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
auti.hr,"HTTPSConnectionPool(host='auti.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'auti.hr' doesn't match 'www.auti.hr'"")))","HTTPSConnectionPool(host='auti.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'auti.hr' doesn't match 'www.auti.hr'"")))"
autoblog.rs,"HTTPSConnectionPool(host='autoblog.rs', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autoblog.rs', port=443): Read timed out. (read timeout=2)"
autoc-one.jp,"HTTPSConnectionPool(host='autoc-one.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autoc-one.jp', port=443): Read timed out. (read timeout=2)"
autofficinall.it,"HTTPConnectionPool(host='www.autofficinall.it', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='autofficinall.it', port=80): Read timed out. (read timeout=2)"
autofficinaorazio.it,"HTTPSConnectionPool(host='autofficinaorazio.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='autofficinaorazio.it', port=80): Read timed out. (read timeout=2)"
autojerry.fi,"HTTPSConnectionPool(host='autojerry.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='autojerry.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
autojournal.cz,"HTTPSConnectionPool(host='autojournal.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autojournal.cz', port=443): Read timed out. (read timeout=2)"
autoportal.hr,"HTTPSConnectionPool(host='autoportal.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autoportal.hr', port=443): Read timed out. (read timeout=2)"
autopower.bg,"HTTPSConnectionPool(host='autopower.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8df7a98790>, 'Connection to autopower.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='autopower.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8dfb073370>, 'Connection to autopower.bg timed out. (connect timeout=2)'))"
autopurkaamot.com,"HTTPSConnectionPool(host='autopurkaamot.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.autopurkaamot.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
autoreviu.lt,"HTTPSConnectionPool(host='autoreviu.lt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8dfa550b80>, 'Connection to autoreviu.lt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='autoreviu.lt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8df4df2760>, 'Connection to autoreviu.lt timed out. (connect timeout=2)'))"
autostation.com,"HTTPSConnectionPool(host='autostation.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'autostation.com' doesn't match 'www.autostation.com'"")))","HTTPSConnectionPool(host='autostation.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'autostation.com' doesn't match 'www.autostation.com'"")))"
autoszektor.hu,"HTTPSConnectionPool(host='autoszektor.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autoszektor.hu', port=443): Read timed out. (read timeout=2)"
avto-fokus.si,"HTTPSConnectionPool(host='avto-fokus.si', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.avto-fokus.si', port=443): Read timed out. (read timeout=2)"
avto.net,Exceeded 30 redirects.,Exceeded 30 redirects.
avtomobilizem.com,"HTTPSConnectionPool(host='avtomobilizem.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='avtomobilizem.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
awilime.com,"HTTPSConnectionPool(host='awilime.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'awilime.com' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net', '*.sso.azurewebsites.net'"")))","HTTPSConnectionPool(host='awilime.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'awilime.com' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net', '*.sso.azurewebsites.net'"")))"
az-gotvia.bg,"HTTPSConnectionPool(host='www.az-gotvia.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.az-gotvia.bg', port=443): Read timed out. (read timeout=2)"
azoo.hr,"HTTPSConnectionPool(host='azoo.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df792ff40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='azoo.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df224c790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
baac.or.th,"HTTPSConnectionPool(host='baac.or.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df8b3a400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='baac.or.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df93f4430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
baahball.com,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
babil.com,"HTTPSConnectionPool(host='babil.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.babil.com', port=443): Read timed out. (read timeout=2)"
babnet.net,"HTTPSConnectionPool(host='babnet.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'babnet.net' doesn't match '*.babnet.net'"")))","HTTPSConnectionPool(host='babnet.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'babnet.net' doesn't match '*.babnet.net'"")))"
backapalankavesti.com,"HTTPSConnectionPool(host='backapalankavesti.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='backapalankavesti.com', port=443): Read timed out. (read timeout=2)"
bacninh.gov.vn,"HTTPSConnectionPool(host='bacninh.gov.vn', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df9b06ac0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='bacninh.gov.vn', port=80): Read timed out. (read timeout=2)"
badenertagblatt.ch,"HTTPSConnectionPool(host='www.badenertagblatt.ch.ch', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df1f80640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPSConnectionPool(host='www.badenertagblatt.ch.ch', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfa91ec70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bahrain.bh,"HTTPSConnectionPool(host='bahrain.bh', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='bahrain.bh', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
balestron.com,"HTTPSConnectionPool(host='balestron.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='balestron.com', port=80): Read timed out. (read timeout=2)"
balix.si,"HTTPSConnectionPool(host='balix.si', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8dfbeb34c0>, 'Connection to balix.si timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='balix.si', port=443): Read timed out. (read timeout=2)"
balkanspress.com,"HTTPSConnectionPool(host='balkanspress.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df2dc72b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='balkanspress.com', port=80): Read timed out. (read timeout=2)"
bancoestado.cl,"HTTPSConnectionPool(host='bancoestado.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfa702f70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bancoestado.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfae64580>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banesco.com,"HTTPSConnectionPool(host='banesco.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8def3a1c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='banesco.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfa64d4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bangkok.go.th,"HTTPSConnectionPool(host='bangkok.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8df28410d0>, 'Connection to bangkok.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='bangkok.go.th', port=80): Read timed out. (read timeout=2)"
bangkokassets.com,"HTTPSConnectionPool(host='www.bangkokassets.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.bangkokassets.com', port=443): Read timed out. (read timeout=2)"
bangkokbank.com,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
bangkokhospital.com,"HTTPSConnectionPool(host='www.bangkokhospital.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.bangkokhospital.com', port=443): Read timed out. (read timeout=2)"
banglapedia.org,"HTTPSConnectionPool(host='banglapedia.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df9f63bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='banglapedia.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df07427f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bankalfalah.com,"HTTPSConnectionPool(host='bankalfalah.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df92eb1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bankalfalah.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfa76c2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banker.bg,"HTTPSConnectionPool(host='www.banker.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.banker.bg', port=443): Read timed out. (read timeout=2)"
bankleumi.co.il,"HTTPSConnectionPool(host='bankleumi.co.il', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df4c9a400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bankleumi.co.il', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfbf3d5e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bankofgreece.gr,"HTTPSConnectionPool(host='bankofgreece.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8df0724b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bankofgreece.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8df6732070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banprogrupopromerica.com.ni,"HTTPSConnectionPool(host='banprogrupopromerica.com.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8dfc52ea60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='banprogrupopromerica.com.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8dfb249850>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banrep.gov.co,"HTTPSConnectionPool(host='banrep.gov.co', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.banrep.gov.co', port=443): Read timed out. (read timeout=2)"
banyuwangikab.go.id,"HTTPSConnectionPool(host='banyuwangikab.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='banyuwangikab.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
